### PR TITLE
Fix incorrect indenting/formatting in ExchangeServiceBase

### PR DESCRIPTION
**Pull request by evpaassen**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@evpaassen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: src/main/java/microsoft/exchange/webservices/data/core/ExchangeServiceBase.java [hunk 1]
```diff
@@ -344,26 +343,25 @@ protected HttpWebRequest prepareHttpWebRequestForUrl(URI url, boolean acceptGzip
...
+      boolean allowAutoRedirect) throws ServiceLocalException, URISyntaxException {
+    // Verify that the protocol is something that we can handle
+    String scheme = url.getScheme();
+    if (!scheme.equalsIgnoreCase(EWSConstants.HTTP_SCHEME)
+        && !scheme.equalsIgnoreCase(EWSConstants.HTTPS_SCHEME)) {
+      String strErr = String.format("Protocol %s isn't supported for service request.", scheme);
+      throw new ServiceLocalException(strErr);
+    }
+
+    if (httpPoolingClient == null)
```
Comment (serious6) [inline src/main/java/microsoft/exchange/webservices/data/core/ExchangeServiceBase.java]: I think google-style suggests using braces when possible.

Comment (evpaassen) [inline src/main/java/microsoft/exchange/webservices/data/core/ExchangeServiceBase.java]: I didn't change that, but you might be correct.

**Comment by evpaassen** [comment]
@serious6 I fixed it.

**Comment by serious6** [comment]
@evpaassen thanks for your contribution

**Pull request merged**